app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java

```diff
@@ -110,6 +110,8 @@ private Measurement getLastMeasurement() {
 	public double getRuntime(){
 		Measurement first = getFirstMeasurement();
 		Measurement last = getLastMeasurement();
+		if (first == null || last  == null)
+			return 0;
```

Comment:
Same as for iOS, please handle this as an exception, not as a valid runtime value.